### ISSUE TITLE
Custom Nodes with text input

### ISSUE THREAD

**Issue by Max-Beier** [open-source-labs/Svelvet#282]
Hey,
adding text input using the native html `<input/>` tag does not seem to work in Svelte v7.0.0. Additionally, it appears that keybindings are not being activated.

Does anyone know of a quick fix to prevent blocking key events?

Cheers.

```svelte
<script lang="ts">
  import { Anchor, Node } from 'svelvet'

  let title = ''
  let description = ''
</script>

<Node bgColor="black" label="StartNode" borderRadius={10}>
  <div class="flex flex-col text-center">
    <section class="text-black resize-none">
      <h1 class="text-white">Start</h1>
      <input
        bind:value={title}
        title="Story Title"
        placeholder="Story Title"
        type="text"
      />
      <input
        bind:value={description}
        title="Story Description"
        placeholder="Story Description"
        type="text"
      />
    </section>
    <div class="absolute m-0 right-0 top-1/2 -translate-y-1/2">
      <Anchor output direction="east" />
    </div>
  </div>
</Node>

```

**Comment by briangregoryholmes** [comment]
Hi, Max. Thanks for bringing this to our attention. I just pushed a new version 7.0.26 that should fix this. We call preventDefault when dragging nodes to stop the browser behavior of text selection on drag. I've added a condition to ensure that the target element is not an input element. I'm going to take a deeper dive to make sure there aren't any similar issues.

Let me know if this works for you when you get the new version installed.
